internal/environment/get_env_value.go

```diff
@@ -0,0 +1,94 @@
+// @license
+// Copyright 2021 Dynatrace LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package environment
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/dynatrace/dynatrace-configuration-as-code/internal/log"
+)
+
+const (
+	DefaultEnvValueInt         = 0
+	ConcurrentRequestsEnvKey   = "CONCURRENT_REQUESTS"
+	DefaultConcurrentDownloads = 50
```

Comment:
with you changes this does not need to be an exported constant anymore. Neither does the Default zero value.
You could make them private to the package, or IMO just write them into the map definition and default return path directly (which would just mean you have assert actual values in the tests)

Reply:
right, asserting in the tests will fix my concern with using the map

Reply:
Done